pages/ksphere/konvoy/1.6/access-authentication/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layout.pug
+navigationTitle: Access and Authentication
+title: Access and Authentication
+menuWeight: 10
+excerpt: Manage cluster security, authentication, and authorization for the Konvoy cluster
+enterprise: false
+---
+
+As Konvoy uses the [Kubernetes security mechanisms].
```

Comment:
This page is going to be updated with more info soon but the PR hasn't yet been merged to master. Seeing as how these are beta docs IMO it's probably fine to leave in for this release.

Reply:
agree. good to merge as is as long as we have plans to make it better before GA.